### DISCUSSION TITLE
Quarkus distribution as default

### DISCUSSION THREAD

**Discussion by stianst** [keycloak/keycloak#8695] [Archive]
As outlined in https://www.keycloak.org/2021/10/keycloak-x-update.html the aim is to switch to the Quarkus distribution as the default distribution in Keycloak 17, and deprecate the WildFly distribution. This proposal outlines how we will make the switch.

* Keycloak.X will be renamed to Keycloak, making the Quarkus distribution the default distribution
* Keycloak (WildFly distribution) will be renamed to Keycloak Legacy, making the WildFly distribution the legacy distribution

h2. Website updates

* Keycloak.X Preview is renamed to Keycloak, and moved to the top of downloads
* Keycloak is renamed to Keycloak Legacy, and moved below Keycloak.X

h2. Documentation

* The current "Server Installation and Configuration" will be renamed to "Server Installation and Configuration Legacy"
* A new "Server Installation and Configuration", which is updated to reflect changes in Quarkus distribution
* Other guides will remain as is, but with providing both alternatives when applicable

h2. Distributions

* Keycloak WildFly distribution is renamed to "keycloak-legacy"
* Keycloak.X distribution is renamed to "keycloak"

h2. Containers

* keycloak/keycloak:17.0.0 will be the Quarkus image
* keycloak/keycloak:17.0.0-legacy will be published for deprecated WildFly distribution, for those unable to make the switch straight away

h2. Operator

The current operator will NOT be renamed and will remain in alpha channel in the OperatorHub.io. It will continue to use the WF distribution.

**Comment by stianst** [comment]
@pedroigor @DGuhr @vmuzikar FIY

**Comment by vmuzikar** [comment]
h2. Operator
The current operator will NOT be renamed and will remain in `alpha` channel in the OperatorHub.io. It will continue to use the WF distribution.

**Reply by stianst** [reply]
Thanks, updated
